src/vm/codeman.cpp

```diff
@@ -5114,7 +5114,7 @@ BOOL NativeImageJitManager::JitCodeToMethodInfo(RangeSection * pRangeSection,
         }
 
         //Get the function entry that corresponds to the real method desc.
-        _ASSERTE(RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry));
+        _ASSERTE((RelativePc >= RUNTIME_FUNCTION__BeginAddress(FunctionEntry)));
```

Comment:
Clang appeared to have trouble expanding the macro correctly, this occured in a couple of places. Perhaps the correct approach would of been to modify the macros however so I could do that instead.